src/database/index.tsx

```diff
@@ -273,6 +273,51 @@ export const getScenarios = async (): Promise<Scenario[]> => {
   }
 };
 
+export const getSharedBaselineScenarios = async (): Promise<Scenario[]> => {
+  try {
+    const db = await getDb();
+
+    const results = await db
+      .collection(scenariosCollectionId)
+      .where(`roles.${currentUserId()}`, "in", ["viewer"])
+      .where("baseline", "==", true)
+      .get();
+
+    const sharedBaselineScenarios = results.docs.map((doc) => {
+      return buildScenario(doc);
+    });
+
+    return sharedBaselineScenarios;
+  } catch (error) {
+    console.error(
+      `Encountered error while attempting to retrieve shared scenarios:`,
+    );
+    console.error(error);
+
+    return [];
+  }
+};
+
+export const getScenarioByStateName = async (
```

Comment:
What if there are multiple scenarios that have facilities matching the state name? I didn't double check the API but doesn't the `find` call below just return the first result found in the `scenarios` array?

Reply:
That's a good point, at the moment this is very closely tied to the context of receiving the `scenarios` param from the return of `getSharedBaselineScenarios`, so I was expecting it to be a collection of baseline scenarios shared with the `covid@recidiviz` user, and from convos with @sychang it seemed like we could assume that only 1 baseline scenario per US State would be available. I could update this to actually return all the results (and rename it to `getScenariosByStateName`) so that it could work in other contexts, and use the first (and presumably only) result in the weekly report context.

Reply:
Looks great, thanks!